src/test/java/seedu/todo/testutil/TaskFactory.java

```diff
@@ -127,4 +128,21 @@ public static ImmutableTask random() {
         
         return tasks;
     }
+
+    //@@author A0135805H
+    /**
+     * Generates a list of random tasks with a random size between {@code lowerBound}
+     * and {@code upperBound} inclusive.
```

Comment:
The use of `nextInt` means it actually excludes the `upperBound` - `list(10, 20)` can produce a list of 10 elements (10 + 0), but never 20 elements (`randInt(10)` will never produce 10)